tests/testthat/test-renderer1-hjust-text-anchor.R

```diff
@@ -79,9 +79,9 @@ test_that("plot_content//table by default", {
   plot_content <- getNodeSet(info$html, '//td[@class="plot_content"]')
   expect_equal(length(plot_content), 1)
   plot_tables <- getNodeSet(info$html, '//td[@class="plot_content"]//table')
-  expect_equal(length(plot_tables), 2)
+  expect_equal(length(plot_tables), 4)
   children <- xmlChildren(plot_content[[1]])
-  expect_equal(length(children), 2)
+  expect_equal(length(children), 1)
 })
```

Comment:
please undo these changes.
the plot_content td should have two children, each one is a ggplot table.